doc/ref/domain.xml

```diff
@@ -605,8 +605,8 @@ The functions <Ref Func="InstallSubsetMaintenance"/>,
 <Ref Func="InstallFactorMaintenance"/> are used to tell &GAP;
 under what conditions an attribute is maintained under taking subsets,
 or forming factor structures or isomorphic domains.
-This is used only when a new attribute is created,
-see&nbsp;<Ref Sect="Creating Attributes and Properties"/>.
+This is used only when a new attribute or property is created,
+see&nbsp;<Ref Sect="NewAttribute"/> and <Ref Sect="NewProperty"/>.
```

Comment:
Is "Sect" the correct reference here? I thought "NewAttribute" was a method so it should be called by "Subsect", but that might not apply in this particular case.

Reply:
It should be `Func`, thank you for catching this.